### ISSUE TITLE
Hard configuration doesn't provide me an admin password

### ISSUE THREAD

**Issue by ilcompratoreconsapevole** [kind-0/nsecbunkerd#13]
### After I entered my nsec the process ends and doesn't provide me an admin password.

When I run `$ npm run nsecbunkerd start` I get this:

```nsecBunker connection string:

bunker://mynpub


❌ Admin interface not starting because no admin npubs were provided
✅ Connected to wss://relay.snort.social
✅ Connected to wss://relay.nsecbunker.com
✅ Connected to wss://nostr.vulpem.com
✅ Connected to wss://relay.damus.io
✅ Connected to wss://nos.lol
✅ nsecBunker ready to serve requests.
📡 5 connected, 0 disconnected, 0 connecting```

How do I resolve it?

**Comment by monty888** [comment]
Same problem for me also.. I noticed that at this point - 
`
npm run nsecbunkerd -- add --name <your-key-name>
`
I don't get the output 

nsecBunker generated an admin password for you:

***************************
So think maybe it bugged out somewhere there

**Comment by pablof7z** [comment]
yeah, I think when I added the web admin stuff I destroyed the CLI admin stuff. Need to revisit this.

**Comment by miggymofongo** [comment]
can confirm that the admin CLI not working for me either. I get that same message with the red x when I try to set it up to get into highlighter.com.

**Comment by jeremycady** [comment]
I had to manually add the admin npub to config/nsecbunker.json, even though I had already added it to the .env.

When I start up the CLI, I get:
Running migrations
❌ no adminNpubs were provided

But then:
✅ Connected to wss://relay.damus.io/
✅ nsecBunker Admin Interface ready
✅ Connected to wss://relay.nsecbunker.com/
🔑 Starting keys []
✅ nsecBunker ready to serve requests.

**Comment by ilcompratoreconsapevole** [comment]
@jeremycady Does it work then, when you have added it? 🤔

You have added it in .env and other file.. would you mind to show us which fields did you fill? Thanks

**Comment by jeremycady** [comment]
My steps to, what appears to be, working CLI:
gh repo clone kind-0/nsecbunkerd
cd nsecbunkerd/
mkdir config
cp .env.example .env
nano .env

Update DATABASE_URL to correct "nsecbunkerd/prisma/nsecbunker.db"
Update ADMIN_NPUBS with npub

npm install
npm run build
npx prisma migrate deploy
npm run nsecbunkerd -- add --name <key-name>
nano config/nsecbunker.json

Add npub to admin.npubs

npm run lfg --admin <npub>

This yields this error at first:
Running migrations
❌ no adminNpubs were provided

Gives me a connection string.

Provides a signing.

Then the following output:
✅ Connected to wss://relay.damus.io/
✅ Connected to wss://relay.nsecbunker.com/
🔑 Starting keys []
✅ nsecBunker ready to serve requests.
✅ nsecBunker Admin Interface ready

I can now log into app.nsecbunker.com and unlock my key, create a policy and a token.

I haven't been able to log into anything using this though.

For example, at coracle.social, when you but in the bunker URL to login, it will give me the following error in the console:
You are not designated to administrate this bunker

I get no feedback from Nostrudel.

I have noticed, in the CLI, every event has "sig: undefined". I'm not sure if that's in an issue or not.

**Comment by jeremycady** [comment]
I forgot to add, I do get a DM when the bunker starts that states:
nsecBunker has started; use bunker://npub*****@relay.nsecbunker.com to connect to it and unlock your key(s)

**Comment by jeremycady** [comment]
Also, I just downloaded and ran the UI locally. 

I've logged into and posted at [nostrudel.ninja](https://nostrudel.ninja/#/)
